<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="user-PC" failures="0" tests="2" name="testcase.LoginTestcasewithExtendReports" time="0.136" errors="1" timestamp="2023-04-19T03:31:21 GST" skipped="1">
  <testcase classname="testcase.LoginTestcasewithExtendReports" name="setup" time="0.135">
    <error message="freemarker/ext/beans/BeansWrapperBuilder" type="java.lang.NoClassDefFoundError">
      <![CDATA[java.lang.NoClassDefFoundError: freemarker/ext/beans/BeansWrapperBuilder
at com.aventstack.extentreports.reporter.BasicFileReporter.start(BasicFileReporter.java:98)
at com.aventstack.extentreports.ReportObservable.register(ReportObservable.java:192)
at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992)
at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762)
at com.aventstack.extentreports.ExtentReports.attachReporter(ExtentReports.java:82)
at testcase.LoginTestcasewithExtendReports.setup(LoginTestcasewithExtendReports.java:29)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:644)
at org.testng.TestRunner.beforeRun(TestRunner.java:633)
at org.testng.TestRunner.run(TestRunner.java:595)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: freemarker.ext.beans.BeansWrapperBuilder
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
... 28 more
]]>
    </error>
  </testcase> <!-- setup -->
  <system-out/>
  <testcase classname="testcase.LoginTestcasewithExtendReports" name="LoginTest" time="0.001">
    <skipped/>
  </testcase> <!-- LoginTest -->
  <system-out/>
</testsuite> <!-- testcase.LoginTestcasewithExtendReports -->
